packages/cli/src/commands/info.ts

```diff
@@ -19,17 +19,30 @@ import os from 'os';
 import { runPlain } from '../lib/run';
 import { paths } from '../lib/paths';
 import { Lockfile } from '../lib/versioning';
+import fs from 'fs-extra';
 
 export default async () => {
   await new Promise(async () => {
     const yarnVersion = await runPlain('yarn --version');
     // eslint-disable-next-line no-restricted-syntax
```

Comment:
Think maybe you can also remove this line too? Not sure why this is valid anymore.

Reply:
For sure, pulled it out